Makefile

```
@@ -54,7 +54,11 @@ build:
 
 .PHONY: protoc
 protoc:
-	find ./rpc/ -name "*.proto" -type f -exec protoc --proto_path=$(GOSRC):. --twirp_out=. --twirp_opt=paths=source_relative --go_out=. --go_opt=paths=source_relative {} \;
+	docker build -f build.Dockerfile -t trivy-build .
```

It takes a few seconds to send build context to Docker daemon, but we don't need it. We can pipe a Dockerfile from STDIN to skip sending.

Before:
```
docker build -f build.Dockerfile -t trivy-build .
Sending build context to Docker daemon  380.8MB
```

After
```
docker build -t trivy-build - < build.Dockerfile
Sending build context to Docker daemon   2.56kB
```

380.8MB -> 2.56KB

---

`trivy-build` sounds like a target building the Trivy binary. Can we rename it to `trivy-protoc`?

```suggestion
	docker build -f build.Dockerfile -t trivy-protoc .
```